Solutions/CohesitySecurity/Playbooks/Cohesity_Send_Incident_Email/azuredeploy.json

```diff
@@ -0,0 +1,187 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Incident Email",
+        "description": "This playbook sends an email to the recipient with the details related to the incidents",
+        "prerequisites": "None",
+        "postDeployment": [],
```

Comment:
Not seeing that all Entries related to metadata has been filled, Kindly fill them

Reply:
Done.